kotsadm/web/src/components/PreflightResultPage.jsx

```diff
@@ -126,10 +133,16 @@ class PreflightResultPage extends Component {
       .then((res) => {
         if (res.status === 200) {
           this.props.data?.refetch();
+        } else {
+          this.setState({
+            errorMessage: `Encountered an error while trying to re-run preflight checks: ${res.error}`
```

Comment:
same here, i don't think there is an `error` field in `res` object. might wanna use the status code.